src/client/opamConfigCommand.mli

```diff
@@ -21,9 +21,9 @@ open OpamStateTypes
     default). [inplace_path] changes how the PATH variable is updated when there
     is already an opam entry: either at the same rank, or pushed in front. *)
 val env:
-  'a global_state -> switch ->
   ?set_opamroot:bool -> ?set_opamswitch:bool ->
   csh:bool -> sexp:bool -> fish:bool -> inplace_path:bool ->
+  'a global_state -> switch ->
```

Comment:
Given that it's just the one place, can we go down the path of greatest caution and just suppress warning 16 in the ML as well?

Reply:
Done